src/styles/main.scss

```diff
@@ -66,10 +66,58 @@ section {
     color: $color-black;
 }
 
+input {
+    padding: 10px 20px 10px 20px;
+    border: none;
+    width: 100%;
+    height: 50px;
+    display: block;
+    border-radius: 10px;
+    min-width: 300px;
+    color: $color-white;
+    font-size: 20px;
+    -webkit-appearance:none;
+    -webkit-box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.45);
```

Comment:
remove all vendor prefixes - this is handled automatically by postcss based on the config we pass it.